cpp/inc/bond/protocol/compact_binary.h

```diff
@@ -61,17 +61,21 @@
                                             '---'---'---'---'---'---'---'---'
                                                                           0
 
+                                            falsy booleans fields are not encoded
```

Comment:
This isn't the case: false boolean fields will be encoded if they are `required`, `required_optional`, or `= nothing`.